### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by patmoore** [nodestream-proj/nodestream#167]
**Describe the bug**
Attempting to install *only* nodestream on an AWS ubuntu box 

**To Reproduce**
% sudo pip install nodestream

Collecting nodestream

  Downloading nodestream-0.9.0-py3-none-any.whl (113 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 113.9/113.9 KB 2.2 MB/s eta 0:00:00

Collecting uvloop<=0.18.0,>=0.17.0

  Downloading uvloop-0.18.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (4.2 MB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.2/4.2 MB 32.4 MB/s eta 0:00:00

Collecting schema<0.8.0,>=0.7.5

  Downloading schema-0.7.5-py2.py3-none-any.whl (17 kB)

Collecting cymple==0.8.1

  Downloading cymple-0.8.1-py2.py3-none-any.whl (11 kB)

Collecting testcontainers[neo4j]<4.0.0,>=3.7.1

  Downloading testcontainers-3.7.1-py2.py3-none-any.whl (45 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 45.3/45.3 KB 5.9 MB/s eta 0:00:00

Collecting psutil<6.0.0,>=5.9.6

  Downloading psutil-5.9.6-cp36-abi3-manylinux_2_12_x86_64.manylinux2010_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (283 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 283.6/283.6 KB 31.5 MB/s eta 0:00:00

Collecting python-json-logger<3.0.0,>=2.0.4

  Downloading python_json_logger-2.0.7-py3-none-any.whl (8.1 kB)

Collecting cleo<3.0.0,>=2.0.1

  Downloading cleo-2.1.0-py3-none-any.whl (78 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 78.7/78.7 KB 7.1 MB/s eta 0:00:00

Collecting confluent-kafka<3.0.0,>=2.3.0

  Downloading confluent_kafka-2.3.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (4.0 MB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.0/4.0 MB 51.2 MB/s eta 0:00:00

Collecting jmespath<2.0.0,>=1.0.1

  Downloading jmespath-1.0.1-py3-none-any.whl (20 kB)

Collecting pandas<3,>=2

  Downloading pandas-2.1.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.3 MB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 12.3/12.3 MB 35.7 MB/s eta 0:00:00

Collecting neo4j<6.0.0,>=5.8.0

  Downloading neo4j-5.15.0.tar.gz (196 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 196.5/196.5 KB 25.4 MB/s eta 0:00:00

  Installing build dependencies ... done

  Getting requirements to build wheel ... done

  Installing backend dependencies ... done

  Preparing metadata (pyproject.toml) ... done

  WARNING: Generating metadata for package neo4j produced metadata for project name unknown. Fix your #egg=neo4j fragments.

Discarding https://files.pythonhosted.org/packages/3f/8a/57cf3ea81791cfc1e1af5b22728bbcd3a0613a18a16fc665835dd9d17f20/neo4j-5.15.0.tar.gz#sha256=5c53eccf5192ac21463d3fac05cf58f1547ed231b953c837aa4393e2fc713ed3 (from https://pypi.org/simple/neo4j/) (requires-python:>=3.7): Requested unknown from[ https://files.pythonhosted.org/packages/3f/8a/57cf3ea81791cfc1e1af5b22728bbcd3a0613a18a16fc665835dd9d17f20/neo4j-5.15.0.tar.gz#sha256=5c53eccf5192ac21463d3fac05cf58f1547ed231b953c837aa4393e2fc713ed3](https://files.pythonhosted.org/packages/3f/8a/57cf3ea81791cfc1e1af5b22728bbcd3a0613a18a16fc665835dd9d17f20/neo4j-5.15.0.tar.gz#sha256=5c53eccf5192ac21463d3fac05cf58f1547ed231b953c837aa4393e2fc713ed3) (from nodestream) has inconsistent name: filename has 'neo4j', but metadata has 'unknown'

  Downloading neo4j-5.14.1.tar.gz (192 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 192.8/192.8 KB 21.1 MB/s eta 0:00:00

  Installing build dependencies ... done

  Getting requirements to build wheel ... done

  Installing backend dependencies ... done

  Preparing metadata (pyproject.toml) ... done

  WARNING: Generating metadata for package neo4j produced metadata for project name unknown. Fix your #egg=neo4j fragments.

Discarding https://files.pythonhosted.org/packages/73/54/9312a287aac523357766797230fe85ec7b70affe9422f05c82b2286d2d47/neo4j-5.14.1.tar.gz#sha256=3200be8550ed16622b2fadf4f749d08b9982989c4bfce6f99ab693f104104cb4 (from https://pypi.org/simple/neo4j/) (requires-python:>=3.7): Requested unknown from https://files.pythonhosted.org/packages/73/54/9312a287aac523357766797230fe85ec7b70affe9422f05c82b2286d2d47/neo4j-5.14.1.tar.gz#sha256=3200be8550ed16622b2fadf4f749d08b9982989c4bfce6f99ab693f104104cb4 (from nodestream) has inconsistent name: filename has 'neo4j', but metadata has 'unknown'

  Downloading neo4j-5.14.0.tar.gz (192 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 192.4/192.4 KB 22.9 MB/s eta 0:00:00

  Installing build dependencies ... done

  Getting requirements to build wheel ... done

  Installing backend dependencies ... done

  Preparing metadata (pyproject.toml) ... done

  WARNING: Generating metadata for package neo4j produced metadata for project name unknown. Fix your #egg=neo4j fragments.

Discarding https://files.pythonhosted.org/packages/30/2c/7429812a750f86b0942067f4be72fc24beabb3079f5d96326293f5219d9f/neo4j-5.14.0.tar.gz#sha256=6040efca47126c01385f09e550fb7d7671b1853a1e1c34908aa3713cebd285da (from https://pypi.org/simple/neo4j/) (requires-python:>=3.7): Requested unknown from https://files.pythonhosted.org/packages/30/2c/7429812a750f86b0942067f4be72fc24beabb3079f5d96326293f5219d9f/neo4j-5.14.0.tar.gz#sha256=6040efca47126c01385f09e550fb7d7671b1853a1e1c34908aa3713cebd285da (from nodestream) has inconsistent name: filename has 'neo4j', but metadata has 'unknown'

  Downloading neo4j-5.13.0.tar.gz (192 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 192.3/192.3 KB 16.6 MB/s eta 0:00:00

  Installing build dependencies ... done

  Getting requirements to build wheel ... done

  Installing backend dependencies ... done

  Preparing metadata (pyproject.toml) ... done

  WARNING: Generating metadata for package neo4j produced metadata for project name unknown. Fix your #egg=neo4j fragments.

Discarding https://files.pythonhosted.org/packages/27/20/b2ada75a6181869fcef6da078681eb12e3c7f1c93e413eb726d961962608/neo4j-5.13.0.tar.gz#sha256=44e60e2b4946340a60b5ba13710d599615f7432612af2bdefc29f44169004de5 (from https://pypi.org/simple/neo4j/) (requires-python:>=3.7): Requested unknown from https://files.pythonhosted.org/packages/27/20/b2ada75a6181869fcef6da078681eb12e3c7f1c93e413eb726d961962608/neo4j-5.13.0.tar.gz#sha256=44e60e2b4946340a60b5ba13710d599615f7432612af2bdefc29f44169004de5 (from nodestream) has inconsistent name: filename has 'neo4j', but metadata has 'unknown'

  Downloading neo4j-5.12.0.tar.gz (190 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 190.9/190.9 KB 22.9 MB/s eta 0:00:00

  Installing build dependencies ... done

  Getting requirements to build wheel ... done

  Installing backend dependencies ... done

  Preparing metadata (pyproject.toml) ... done

  WARNING: Generating metadata for package neo4j produced metadata for project name unknown. Fix your #egg=neo4j fragments.

Discarding https://files.pythonhosted.org/packages/19/11/9af80c8621ba433039bf47acd7b037998138516eb86fdb9ba492c49ee844/neo4j-5.12.0.tar.gz#sha256=00a776a687267150f9e1950017316b751cf63db7b734a699b1405ac20fd4a731 (from https://pypi.org/simple/neo4j/) (requires-python:>=3.7): Requested unknown from https://files.pythonhosted.org/packages/19/11/9af80c8621ba433039bf47acd7b037998138516eb86fdb9ba492c49ee844/neo4j-5.12.0.tar.gz#sha256=00a776a687267150f9e1950017316b751cf63db7b734a699b1405ac20fd4a731 (from nodestream) has inconsistent name: filename has 'neo4j', but metadata has 'unknown'

  Downloading neo4j-5.11.0.tar.gz (188 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 188.4/188.4 KB 24.1 MB/s eta 0:00:00

  Installing build dependencies ... done

  Getting requirements to build wheel ... done

  Installing backend dependencies ... done

  Preparing metadata (pyproject.toml) ... done

  WARNING: Generating metadata for package neo4j produced metadata for project name unknown. Fix your #egg=neo4j fragments.

Discarding https://files.pythonhosted.org/packages/d0/68/f9ce3c2e8c4e4fa5e175dffbda7a920acb59e3db97d9efa85d0e3c671758/neo4j-5.11.0.tar.gz#sha256=81d425ef9a53279c6909ec8d33e7dc913acc840292f0f3a047f3c3c5b74bccb5 (from https://pypi.org/simple/neo4j/) (requires-python:>=3.7): Requested unknown from https://files.pythonhosted.org/packages/d0/68/f9ce3c2e8c4e4fa5e175dffbda7a920acb59e3db97d9efa85d0e3c671758/neo4j-5.11.0.tar.gz#sha256=81d425ef9a53279c6909ec8d33e7dc913acc840292f0f3a047f3c3c5b74bccb5 (from nodestream) has inconsistent name: filename has 'neo4j', but metadata has 'unknown'

  Downloading neo4j-5.10.0.tar.gz (187 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 187.7/187.7 KB 21.6 MB/s eta 0:00:00

  Installing build dependencies ... done

  Getting requirements to build wheel ... done

  Installing backend dependencies ... done

  Preparing metadata (pyproject.toml) ... done

  WARNING: Generating metadata for package neo4j produced metadata for project name unknown. Fix your #egg=neo4j fragments.

Discarding https://files.pythonhosted.org/packages/21/53/24d8ba3afffd91c59f356fffbf5d0e2afdc9085ee40927b16d9ee65f05a6/neo4j-5.10.0.tar.gz#sha256=90616b2de1dc5a5e0ab17b72f0762f3496ac9f21c58bc0067f44d40c41db6a4e (from https://pypi.org/simple/neo4j/) (requires-python:>=3.7): Requested unknown from https://files.pythonhosted.org/packages/21/53/24d8ba3afffd91c59f356fffbf5d0e2afdc9085ee40927b16d9ee65f05a6/neo4j-5.10.0.tar.gz#sha256=90616b2de1dc5a5e0ab17b72f0762f3496ac9f21c58bc0067f44d40c41db6a4e (from nodestream) has inconsistent name: filename has 'neo4j', but metadata has 'unknown'

  Downloading neo4j-5.9.0.tar.gz (188 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 188.5/188.5 KB 24.7 MB/s eta 0:00:00

  Installing build dependencies ... done

  Getting requirements to build wheel ... done

  Installing backend dependencies ... done

  Preparing metadata (pyproject.toml) ... done

  WARNING: Generating metadata for package neo4j produced metadata for project name unknown. Fix your #egg=neo4j fragments.

Discarding https://files.pythonhosted.org/packages/e9/b7/faa4628ea379f9e43772ca4448356a35e58ce44306a91117f2892487e187/neo4j-5.9.0.tar.gz#sha256=b0abc0065f616bfd8230a48b1f6c91f9aacd7e2aea76d72d09a745ae169cf4da (from https://pypi.org/simple/neo4j/) (requires-python:>=3.7): Requested unknown from https://files.pythonhosted.org/packages/e9/b7/faa4628ea379f9e43772ca4448356a35e58ce44306a91117f2892487e187/neo4j-5.9.0.tar.gz#sha256=b0abc0065f616bfd8230a48b1f6c91f9aacd7e2aea76d72d09a745ae169cf4da (from nodestream) has inconsistent name: filename has 'neo4j', but metadata has 'unknown'

  Downloading neo4j-5.8.1.tar.gz (187 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 187.7/187.7 KB 21.4 MB/s eta 0:00:00

  Installing build dependencies ... done

  Getting requirements to build wheel ... done

  Installing backend dependencies ... done

  Preparing metadata (pyproject.toml) ... done

  WARNING: Generating metadata for package neo4j produced metadata for project name unknown. Fix your #egg=neo4j fragments.

Discarding https://files.pythonhosted.org/packages/97/9f/8f04fe1e04c832d87283ede166290cc29c1c3aa0d274946f88361d42a9d3/neo4j-5.8.1.tar.gz#sha256=79c947f402e9f8624587add7b8af742b38cbcdf364d48021c5bff9220457965b (from https://pypi.org/simple/neo4j/) (requires-python:>=3.7): Requested unknown from https://files.pythonhosted.org/packages/97/9f/8f04fe1e04c832d87283ede166290cc29c1c3aa0d274946f88361d42a9d3/neo4j-5.8.1.tar.gz#sha256=79c947f402e9f8624587add7b8af742b38cbcdf364d48021c5bff9220457965b (from nodestream) has inconsistent name: filename has 'neo4j', but metadata has 'unknown'

  Downloading neo4j-5.8.0.tar.gz (187 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 187.5/187.5 KB 18.7 MB/s eta 0:00:00

  Installing build dependencies ... done

  Getting requirements to build wheel ... done

  Installing backend dependencies ... done

  Preparing metadata (pyproject.toml) ... done

  WARNING: Generating metadata for package neo4j produced metadata for project name unknown. Fix your #egg=neo4j fragments.

Discarding https://files.pythonhosted.org/packages/6a/86/8045ef105ecbc1069d36856e0fe52b0f9b7838a55277908034b4bef0f9dc/neo4j-5.8.0.tar.gz#sha256=af5e70801f63eaaff8fa90bfdc943c144fb6857438590782b30d0ed745229b9e (from https://pypi.org/simple/neo4j/) (requires-python:>=3.7): Requested unknown from[ https://files.pythonhosted.org/packages/6a/86/8045ef105ecbc1069d36856e0fe52b0f9b7838a55277908034b4bef0f9dc/neo4j-5.8.0.tar.gz#sha256=af5e70801f63eaaff8fa90bfdc943c144fb6857438590782b30d0ed745229b9e](https://files.pythonhosted.org/packages/6a/86/8045ef105ecbc1069d36856e0fe52b0f9b7838a55277908034b4bef0f9dc/neo4j-5.8.0.tar.gz#sha256=af5e70801f63eaaff8fa90bfdc943c144fb6857438590782b30d0ed745229b9e) (from nodestream) has inconsistent name: filename has 'neo4j', but metadata has 'unknown'

Collecting nodestream

  Downloading nodestream-0.8.20-py3-none-any.whl (108 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 108.5/108.5 KB 14.3 MB/s eta 0:00:00

  Downloading nodestream-0.8.19-py3-none-any.whl (108 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 108.5/108.5 KB 14.5 MB/s eta 0:00:00

  Downloading nodestream-0.8.18-py3-none-any.whl (107 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 107.8/107.8 KB 13.1 MB/s eta 0:00:00

  Downloading nodestream-0.8.17-py3-none-any.whl (107 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 107.7/107.7 KB 16.6 MB/s eta 0:00:00

  Downloading nodestream-0.8.16-py3-none-any.whl (107 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 107.7/107.7 KB 15.0 MB/s eta 0:00:00

  Downloading nodestream-0.8.15-py3-none-any.whl (107 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 107.6/107.6 KB 16.6 MB/s eta 0:00:00

Collecting aiokafka<0.9.0,>=0.8.0

  Downloading aiokafka-0.8.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.1 MB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 54.1 MB/s eta 0:00:00

Collecting nodestream

  Downloading nodestream-0.8.14-py3-none-any.whl (107 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 107.4/107.4 KB 15.8 MB/s eta 0:00:00

  Downloading nodestream-0.8.13-py3-none-any.whl (106 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 106.7/106.7 KB 14.3 MB/s eta 0:00:00

  Downloading nodestream-0.8.12-py3-none-any.whl (106 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 106.6/106.6 KB 7.0 MB/s eta 0:00:00

  Downloading nodestream-0.8.11-py3-none-any.whl (106 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 106.6/106.6 KB 8.4 MB/s eta 0:00:00

  Downloading nodestream-0.8.10-py3-none-any.whl (106 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 106.5/106.5 KB 9.7 MB/s eta 0:00:00

  Downloading nodestream-0.8.9-py3-none-any.whl (106 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 106.6/106.6 KB 7.6 MB/s eta 0:00:00

  Downloading nodestream-0.8.8-py3-none-any.whl (106 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 106.5/106.5 KB 14.4 MB/s eta 0:00:00

  Downloading nodestream-0.8.7-py3-none-any.whl (106 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 106.4/106.4 KB 15.9 MB/s eta 0:00:00

  Downloading nodestream-0.8.6-py3-none-any.whl (107 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 107.1/107.1 KB 16.3 MB/s eta 0:00:00

Collecting jq<2.0.0,>=1.4.1

  Downloading jq-1.6.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (656 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 656.0/656.0 KB 48.3 MB/s eta 0:00:00

Collecting nodestream

  Downloading nodestream-0.8.5-py3-none-any.whl (107 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 107.1/107.1 KB 15.0 MB/s eta 0:00:00

  Downloading nodestream-0.8.4-py3-none-any.whl (106 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 106.8/106.8 KB 16.1 MB/s eta 0:00:00

  Downloading nodestream-0.8.3-py3-none-any.whl (106 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 106.8/106.8 KB 16.5 MB/s eta 0:00:00

  Downloading nodestream-0.8.2-py3-none-any.whl (113 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 113.5/113.5 KB 18.8 MB/s eta 0:00:00

  Downloading nodestream-0.8.1-py3-none-any.whl (113 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 113.1/113.1 KB 18.3 MB/s eta 0:00:00

  Downloading nodestream-0.7.2-py3-none-any.whl (110 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 110.3/110.3 KB 18.4 MB/s eta 0:00:00

  Downloading nodestream-0.7.1-py3-none-any.whl (110 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 110.2/110.2 KB 13.5 MB/s eta 0:00:00

  Downloading nodestream-0.7.0-py3-none-any.whl (110 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 110.2/110.2 KB 18.2 MB/s eta 0:00:00

  Downloading nodestream-0.6.1-py3-none-any.whl (109 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 109.9/109.9 KB 17.4 MB/s eta 0:00:00

  Downloading nodestream-0.6.0-py3-none-any.whl (109 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 109.9/109.9 KB 16.9 MB/s eta 0:00:00

  Downloading nodestream-0.5.0-py3-none-any.whl (106 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 106.0/106.0 KB 16.3 MB/s eta 0:00:00

  Downloading nodestream-0.4.0-py3-none-any.whl (103 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 103.2/103.2 KB 9.3 MB/s eta 0:00:00

  Downloading nodestream-0.3.3-py3-none-any.whl (101 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 101.8/101.8 KB 14.0 MB/s eta 0:00:00

  Downloading nodestream-0.3.2-py3-none-any.whl (101 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 101.8/101.8 KB 16.3 MB/s eta 0:00:00

  Downloading nodestream-0.3.1-py3-none-any.whl (101 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 101.8/101.8 KB 5.9 MB/s eta 0:00:00

  Downloading nodestream-0.3.0-py3-none-any.whl (101 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 101.8/101.8 KB 15.9 MB/s eta 0:00:00

  Downloading nodestream-0.2.0-py3-none-any.whl (98 kB)

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 98.5/98.5 KB 13.9 MB/s eta 0:00:00

ERROR: Cannot install nodestream==0.2.0, nodestream==0.3.0, nodestream==0.3.1, nodestream==0.3.2, nodestream==0.3.3, nodestream==0.4.0, nodestream==0.5.0, nodestream==0.6.0, nodestream==0.6.1, nodestream==0.7.0, nodestream==0.7.1, nodestream==0.7.2, nodestream==0.8.1, nodestream==0.8.10, nodestream==0.8.11, nodestream==0.8.12, nodestream==0.8.13, nodestream==0.8.14, nodestream==0.8.15, nodestream==0.8.16, nodestream==0.8.17, nodestream==0.8.18, nodestream==0.8.19, nodestream==0.8.2, nodestream==0.8.20, nodestream==0.8.3, nodestream==0.8.4, nodestream==0.8.5, nodestream==0.8.6, nodestream==0.8.7, nodestream==0.8.8, nodestream==0.8.9 and nodestream==0.9.0 because these package versions have conflicting dependencies.

 

The conflict is caused by:

    nodestream 0.9.0 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.20 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.19 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.18 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.17 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.16 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.15 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.14 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.13 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.12 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.11 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.10 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.9 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.8 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.7 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.6 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.5 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.4 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.3 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.2 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.8.1 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.7.2 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.7.1 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.7.0 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.6.1 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.6.0 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.5.0 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.4.0 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.3.3 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.3.2 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.3.1 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.3.0 depends on neo4j<6.0.0 and >=5.8.0

    nodestream 0.2.0 depends on neo4j<6.0.0 and >=5.8.0

 

To fix this you could try to:

1. loosen the range of package versions you've specified

2. remove package versions to allow pip attempt to solve the dependency conflict

 

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts

**Expected behavior**
1. I would expect there to be an option to *not* install neo4j. 
2. I would expect the failure of neo4j installation to not impact the installation of nodestream 

**Additional context**
I am planning on installing nodestream on a separate AWS box than that of the neo4j instance.

**Comment by zprobst** [comment]
This is an occurence of #190. Closing in favor of that.
